src/Cake.DotNetTool.Module/DotNetToolContentResolver.cs

```diff
@@ -73,6 +73,10 @@ public IReadOnlyCollection<IFile> GetFiles(PackageReference package, PackageType
                         return GetToolFiles(new DirectoryPath(_environment.GetEnvironmentVariable("USERPROFILE")).Combine(".dotnet/tools"), package);
                     }
                 }
+                else if (package.Parameters.ContainsKey("local"))
+                {
+                    return new List<IFile>(); // TODO: implementation

```

Comment:
This is the hard part. Anyone got any suggestions?

Reply:
This is a good question, getting and executing tools from global  NuGet cache is possible quick PoC gist here
https://gist.github.com/devlead/559d9b082c9cd81c5c5d52f94d68380c

Don't know if that helps, but potentially one could return entry points 🤔